container-search/src/main/java/com/yahoo/search/ranking/GlobalPhaseRanker.java

```diff
@@ -46,9 +46,10 @@ public void process(Query query, Result result, String schema) {
             }
             return simple;
         };
-        // TODO need to get rerank-count somehow
-        int rerank = 7;
-        ResultReranker.rerankHits(result, new HitRescorer(supplier), rerank);
+        int rerankCount = data.rerankCount();
+        if (rerankCount < 0)
```

Comment:
Consider moving the default rerankCount to `config-model`

Reply:
it's probably easier to find if it's here, I'll keep it as-is for now.